rootdir/vendor/etc/camera/camera_config.xml

```diff
@@ -23,6 +23,30 @@
       <MinFocusDistance>0.1</MinFocusDistance>
     </LensInfo>
   </CameraModuleConfig>
+  <CameraModuleConfig>
+    <CameraId>1</CameraId>
+    <SensorName>s5k4h8</SensorName>
+    <ActuatorName>lc898214xc</ActuatorName>
+    <FlashName>pmic</FlashName>
+    <ChromatixName>s5k4h8_chromatix</ChromatixName>
+    <ModesSupported>1</ModesSupported>
+    <Position>BACK_AUX</Position>
+    <MountAngle>90</MountAngle>
```

Comment:
Either here or in kernel

Reply:
????

Reply:
You have changed it in your kernel DT (not in any pr), so you either change it here or there :p

Reply:
Oh so not in both ?

Reply:
@kholk I think this is how you meant it ?